### DISCUSSION TITLE
Zoom screen sharing integration

### DISCUSSION THREAD

**Discussion by ITMicaH** [sircharlo/meeting-media-manager#271] [Ideas] [answered]
It would be great if Zoom Screen sharing could start after the first time you click the play button. I haven't found a way to do this using the Zoom api but a simple Alt+S keystroke would be great. Ofcourse you'd have to find a way to check if the Meeting Controls window is already open otherwise sending Alt+S would stop the active screen share, but that should be doable.

**Comment by sircharlo** [comment, marked as answer]
I suggest looking into OBS Studio, a free app. The next release of JWMMF will have OBS integration features.
OBS can be set up to "output" a virtual camera, which can then be selected in Zoom as the camera. 

JWMMF will integrate with OBS to indicate which "scenes" are active (show speaker/stage camera, show media window, show media window with stage view overlay, etc.). It's a very versatile little app, and I think it's the easiest way to share media at the KH and over Zoom simultaneously, as Zoom screen sharing becomes completely unnecessary.

**Reply by sircharlo** [reply]
The reason why I went this way is that Zoom is notoriously hard to control from other apps, unfortunately.

**Reply by ITMicaH** [reply]
I get that, but using OBS is yet another tool to learn for all our video-operators. And they'd have to create the scenes in OBS manually before each meeting. The reason why we want to use your tool is that everything is in one place: auto-download from jw.org, add custom images and present om the external monitor. If the upcoming OBS integration can create the scenes automatically we'll definitely look into that but for now this defeats the purpose of the tool for us, which is to make it easier to download and present images/video.

**Reply by ITMicaH** [reply]
Sorry, I just figured out you'd only need one scene with the media window. I'll look into it once the integration is ready. Thanks for all you hard work.

**Reply by sircharlo** [reply]
> Sorry, I just figured out you'd only need one scene with the media window. I'll look into it once the integration is ready. Thanks for all you hard work.

Exactly. Also, the scenes only need to be set up **once** in OBS by the local IT brother. Then they are always accessible for everyone in JWMMF, as long as OBS is running.

**Reply by sircharlo** [reply]
Take a look at the updated [documentation](https://sircharlo.github.io/jw-meeting-media-fetcher/#/present-media), hopefully it helps with the setup now that the latest version is out.
